### PR TITLE
improve lint code

### PR DESCRIPTION
**Pull request by hirohaji**
Fixing issues on #35 about code lint

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -6,8 +6,10 @@ const app = express()
...
 app.use('/api/v1', cors(), router)
 
-const port = 8001
+// const port = 8001
 
+/*
 app.listen(port, () =>
   console.log(`backend service listening at port ${port}`)
 )
+*/
```
Comment (paulodiovani) [inline backend/index.js]: I don't get why comment this out.
All these lines are very important.

File: backend/services/PullRequest.js [hunk 2]
```diff
@@ -17,24 +17,25 @@ class PullRequest {
...
-      let response = await axios.get(pEndpoint(this.startDate, this.endDate))
+      const response = await axios.get(pEndpoint(this.startDate, this.endDate))
 
       this.data = response.data
 
       return this
     } catch (error) {
-      console.error(error)
+      return error
+      // console.error(error)
```
Comment (paulodiovani) [inline backend/services/PullRequest.js]: Let's use `debug` module here.

```js
const debug = require('debug')(services:pull-request)

// and then...

debug(error)
```

Must also install the module: `npm install --save debug`

File: backend/workers/fetch-pull-requests.js [hunk 3]
```diff
@@ -76,7 +76,7 @@ async function runCron (startDate, endDate) {
 
     return redisClient
   } catch (error) {
-    console.log(error)
+    // console.log(error)
```
Comment (paulodiovani) [inline backend/workers/fetch-pull-requests.js]: Another place to use `debug` module.
In this case import as:

```js
const debug = require('debug')('workers:fetch-pull-requests')

File: tests/unit/backend/services/PullRequest.spec.js [hunk 4]
```diff
@@ -26,7 +26,8 @@ describe('PullRequest service', () => {
       try {
         await pr.getAll()
       } catch (error) {
-        console.error(error)
+        return error
+        // console.error(error)
```
Comment (paulodiovani) [inline tests/unit/backend/services/PullRequest.spec.js]: Another one:

```js
const debug = require('debug')('services:pull-request')

**Review by paulodiovani — CHANGES_REQUESTED** [review]
So far so good, :)

Please check the comments.

**Comment by paulodiovani** [comment]
Hi @hirohaji  ,
We have just a couple more days to finish the challenge.
Do you have some time to do this changes? :running_man:

**Comment by hirohaji** [comment]
Hi, sorry for the delay, I will review my commit. I commented the console logs because they generate warnings.

File: backend/index.js (hunk 1 above)
Comment (hirohaji) [reply backend/index.js]: Do you want me to put this log as a debug too? The log throws a warning on the lint

File: tests/unit/backend/services/PullRequest.spec.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import PullRequest from '../../../../backend/services/PullRequest'
 import axios from 'axios'
 import mockedResponse from '../../support/response.json'
+const debug = require('debug')('services:pull-request')
```
Comment (paulodiovani) [inline tests/unit/backend/services/PullRequest.spec.js]: We don't need this in tests.

File: tests/unit/backend/services/PullRequest.spec.js [hunk 6]
```diff
@@ -26,7 +27,8 @@ describe('PullRequest service', () => {
       try {
         await pr.getAll()
       } catch (error) {
-        console.error(error)
+        debug(error)
+        // console.error(error)
       }
```
Comment (paulodiovani) [inline tests/unit/backend/services/PullRequest.spec.js]: I didn't notice before this was a test.
Please, remove the `try / catch` and leave only `await pr.getAll()`
￼
If this makes the test fails, you can probably remove it as well.

File: backend/services/PullRequest.js [hunk 7]
```diff
@@ -17,24 +18,25 @@ class PullRequest {
...
     try {
-      let response = await axios.get(pEndpoint(this.startDate, this.endDate))
+      const response = await axios.get(pEndpoint(this.startDate, this.endDate))
 
       this.data = response.data
 
       return this
     } catch (error) {
-      console.error(error)
+      // console.error(error)
```
Comment (paulodiovani) [inline backend/services/PullRequest.js]: Please, also remove this commented line

File: backend/workers/fetch-pull-requests.js [hunk 8]
```diff
@@ -76,7 +77,8 @@ async function runCron (startDate, endDate) {
...
```
Comment (paulodiovani) [inline backend/workers/fetch-pull-requests.js]: Please, also remove this commented line

**Review by paulodiovani — COMMENTED** [review]
Just some more minor changes and we are good to go.

**Review by paulodiovani — APPROVED** [review]

**Pull request merged**